protocol/messenger_mailserver.go

```diff
@@ -22,7 +22,7 @@ import (
 
 const (
 	initialStoreNodeRequestPageSize = 4
-	defaultStoreNodeRequestPageSize = 20
+	defaultStoreNodeRequestPageSize = 50
```

Comment:
Why not make it `90` closer to `100` which is max size? 
Or we need to tune this based on performance

Reply:
I wanted to see how this behaves with 50, and see if it has an impact on the storenodes as well as seeing if it affects negatively the UX since it means that now instead of a possible maximum of 20mb of transfer in a single request, we could get 50mb (Maybe this happen with the content topic for the community descriptions?). Assuming all is good we could increase it to 100.